core-java-modules/core-java-collections-5/src/test/java/com/baeldung/immutables/ImmutableCollectionsUnitTest.java

```diff
@@ -0,0 +1,54 @@
+package com.baeldung.immutables;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+
+public class ImmutableCollectionsUnitTest {
```

Comment:
Please remove class and method public modifiers they are not needed.

Reply:
Done @JavierPeris